lib/file_management.js

```diff
@@ -0,0 +1,124 @@
+const archethic = require('archethic')
+const chalk = require('chalk')
+const crypto = require('crypto')
+const fs = require('fs')
+const path = require('path')
+const jsdom = require("jsdom")
+const {
+    JSDOM
+} = jsdom
+const mime = require('mime')
+
+function confirmationCallback(endpoint, address, message, file) {
+    return (nbConfirmations) => {
+        if (nbConfirmations == 1) {
+            console.log(chalk.green(message))
+            console.log(chalk.blue(endpoint + "/api/last_transaction/" + address + "/content?mime=" + mime.getType(file)))
+        }
+        console.log(chalk.magenta("Transaction confirmed with " + nbConfirmations + " replications"))
+    }
+
+}
+
+
+module.exports = {
+
+    hmac: function (Files, algo, myseed, Seed, Address) {
```

Comment:
Those functions are not clear about what they are doing.
You should not refactor just to refactor but help to reader to understand what your code is doing, here it's not clear